processor/src/main/java/com/instagram/common/json/annotation/processor/JsonAnnotationProcessor.java

```diff
@@ -41,12 +37,16 @@
  * This annotation processor is run at compile time to find classes annotated with {@link JsonType}.
  * Deserializers are generated for such classes.
  */
+@SupportedOptions({"generateSerializers"})
```

Comment:
mildly concerned that `generateSerializers` is a very generic name, but probably not a big deal.